lib/kwtool_web/templates/keyword/show.html.eex

```diff
@@ -0,0 +1,85 @@
+<h1>Result for "<%= @keyword.phrase %>"  <span class="badge status--added"><%= @keyword.status %></span></h1>
+
+<div class="keyword-details">
+  <ul class="keyword-details__list">
+    <li>
+      <div class="row">
+        <div class="col-7">
+          <i class="far fa-gem"></i>
```

Comment:
How about extracting these row/column/icon to the stylesheets. We could reduce some repetitions here. What do you think?

Reply:
yes, that's the way to go, I am working on it now, however there is some issue with the webpack, when try I to include the font-awesome icon library into the webpack loader.

Reply:
I finished refactoring this code. Please help me check again. Thanks!